src/e2e/scala/io/github/paoloboni/FapiE2ETests.scala

```diff
@@ -65,6 +67,19 @@ class FapiE2ETests
           quantity = 10
         )
       )
+    orderIdTest = response.unsafeRunSync()(runtime).orderId
+    response.asserting(_ shouldBe a[FutureOrderCreateResponse])
+  }
+
+
+  "getOrder" in { client =>
+    client
+      .getOrder(
+        FutureGetOrderParams.OrderId(
+          symbol = "LTCUSDT",
+          orderId = orderIdTest
```

Comment:
rather than using `orderIdTest` from the previous test case, I'd suggest to create an order as part of this test case. Something like:
```scala
  "getOrder" in { client =>
    val side = Random.shuffle(OrderSide.values).head
    val result = for {
      orderCreated <- client
        .createOrder(
          FutureOrderCreateParams.MARKET(
            symbol = "LTCUSDT",
            side = side,
            positionSide = FuturePositionSide.BOTH,
            quantity = 10
          )
        )
      orderFetched <- client
        .getOrder(
          FutureGetOrderParams.OrderId(
            symbol = "LTCUSDT",
            orderId = orderCreated.orderId
          )
        )
    } yield orderFetched
    result.asserting(_ shouldBe a[FutureOrderCreateResponse]) // this cannot be a FutureOrderCreateResponse - needs a different data structure
  }
```